Sources/Common/Extensions/URLExtension.swift

```diff
@@ -91,28 +91,54 @@ extension URL {
         public static let blob = NavigationalScheme(rawValue: "blob")
         public static let about = NavigationalScheme(rawValue: "about")
 
+        public static let mailto = NavigationalScheme(rawValue: "mailto")
```

Comment:
Are these changes related?

Reply:
mailto uses the same user@domain.com URL structure which was returning false for url.isValid for punycoded URLs